modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/plugins/fields-sidebar/components/FieldsSidebarSettingsHeader.js

```diff
@@ -39,6 +39,7 @@ export default function FieldsSidebarSettingsHeader() {
 				monospaced={false}
 				onClick={() => {
 					dispatch({
+						submitButtonId,
```

Comment:
Since `submitButtonId` is a configuration you can retrieve this information directly into the reducer.